README.md

```diff
@@ -1,57 +1,72 @@
-# Run MATLAB® Tests
+# Action for Running MATLAB Tests
 
-Use this action to run your MATLAB and Simulink® tests and generate artifacts
-such as JUnit test results and Cobertura code coverage reports. To run the tests
-in your project, the Run MATLAB Tests action uses the first MATLAB version on
-the runner's system `PATH`.
+The [Run MATLAB Tests](#run-matlab-tests) GitHub&reg; action enables you to run MATLAB&reg; and Simulink&reg; tests and generate artifacts such as JUnit test results and Cobertura code coverage reports. You can run tests and generate artifacts on a [GitHub-hosted](https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners) or [self-hosted](https://docs.github.com/en/free-pro-team@latest/actions/hosting-your-own-runners/about-self-hosted-runners) runner:
 
-**Note**: By running the code in this action, you will be executing third-party
-code that is licensed under separate terms.
+- If you want to use a self-hosted runner, you must set up a computer with MATLAB (R2013b or later) as your runner. The action uses the first MATLAB version on the runner's path.
```

Comment:
"The action uses the first MATLAB version on the runner's **system** path."

Reply:
Good suggestion. Done!